### PR TITLE
Support welcome email

### PR DESCRIPTION
**Pull request by ben181231**
connect #10

### CODE REVIEW

File: forgot_password/handlers/welcome_email.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            smtp_host=smtp_settings.host,
+            smtp_port=smtp_settings.port,
+            smtp_mode=smtp_settings.mode,
+            smtp_login=smtp_settings.login,
+            smtp_password=smtp_settings.password,
+        )
+        mailer.send_mail(sender, user.email, subject, text,
+                         html=html, reply_to=reply_to)
+        logger.info('Successfully sent welcome email '
+                    'to user.')
```
Comment (cheungpat) [inline forgot_password/handlers/welcome_email.py]: which user? (include user id)

File: forgot_password/handlers/welcome_email.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    except Exception as ex:
+        logger.error('An error occurred sending welcome '
```
Comment (cheungpat) [inline forgot_password/handlers/welcome_email.py]: if you call `logger.exception` the exception is printed so you don’t need to format the exception here.

also need to know which user id

File: forgot_password/handlers/welcome_email.py [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+        #  No need to register
+        return
+
+    @skygear.after_save('user', async=True)
+    def user_after_save(record, original_record, db):
+        if original_record:
+            # ignore for old users
+            return
+
+        # FIXME: remove this when skygear maintain the same session between
```
Comment (cheungpat) [inline forgot_password/handlers/welcome_email.py:111]: include skygear-server issue number here

**Review by cheungpat — CHANGES_REQUESTED** [review]

**Comment by ben181231** [comment]
Updated.

**Pull request merged**